packages/cli/src/collect/collect.js

```diff
@@ -80,6 +80,12 @@ function buildCommand(yargs) {
       default: 3,
       type: 'number',
     },
+    numberOfFiles: {
```

Comment:
WDYT about...

`maxAutodiscoverUrls`
`maxStaticDistDirPages`
`maxNumberOfPages`

any of those sound good to you? anything that conveys the narrower scope (just applies to static dist dir automatic detection fallback URLs, not URLs or files in general) sgtm

Reply:
I think maxAutodiscoverUrls is best,
the other ones could be confusing because you technically can add more urls from the staticDistDir manually,